fighters/common/src/function_hooks/transition.rs

```diff
@@ -1,6 +1,14 @@
 use super::*;
 use globals::*;
 
+
+pub fn get_fighter_common_from_accessor<'a>(boma: &'a mut BattleObjectModuleAccessor) -> &'a mut L2CFighterCommon {
```

Comment:
why is this just randomly shoved here?
This should be in utils or something, though im pretty sure it was mentioned that there was a built-in call for this already.

cc @blu-dev

Reply:
Was added there temporarily until @blu-dev 's tag PR gets merged (it's included in utils there)

Will make another commit when that gets merged